docs/source/conf.py

```diff
@@ -155,3 +161,13 @@ def setup(sphinx):
 
 # -- Options for EPUB output
 epub_show_urls = "footnote"
+
+# -- Automatic fetch citation info from zenodo --------------------------------
+bibtex = get_latest_jupedsim_bibtex(version)
+logging.info("Bibtex fetched successfully.")
```

Comment:
import logging missing

Reply:
removed the logging (was for debugging anyways)

Reply:
Hmm, since this is running in the CI and hence errors are notoriously hard to debug I would actually argue for enabling debug output so that we can as clearly as possible diagnose issues.